### DISCUSSION TITLE
Automatically load preset for specific active app?

### DISCUSSION THREAD

**Discussion by Mr-Foxtrot** [maximmax42/Discord-CustomRP#167] [Ideas]
So far, i've not seen a single rich presence app _this_ well polished: so I want to bring forward a idea that would probably make this the best rich presence tool around.

An option to set a preset to load for specific apps.
Let you create a preset with a sort of whitelist of window titles, and have the software try to load that preset when a window title or exe name is found amongst that list. This way, for apps that don't have rich presence you can sorta fake it using this tool. 

My apologies if this isn't the best wording for it but I hope you understand what I mean.

**Comment by maximmax42** [comment]
Hmm. Good idea, not sure about execution.
